### DISCUSSION TITLE
Swedish Translation

### DISCUSSION THREAD

**Discussion by Olle-Mineur** [TandoorRecipes/recipes#551] [General]
Hi!
I would want to begin the translation to swedish, but I did not find how to contact to start a new translation on https://translate.tandoor.dev/ so I started a discussion here on Github. Thank you anyways and keep up the good work with this project!

**Comment by vabene1111** [comment]
![image](https://user-images.githubusercontent.com/6819595/114317973-fce86900-9b0a-11eb-9b80-bb973ba2be6f.png)

is that button not available for you ? 

If so i will need to find out how to enable it .. whatever the case is, i added Swedish for you

**Reply by Olle-Mineur** [reply]
I can't see that button on my screen, but I sent you a screenshot how I see the page.
![image](https://user-images.githubusercontent.com/48391622/114318267-a54afd00-9b0c-11eb-9904-d0631735fea0.png)

Thank you for adding Swedish!

**Reply by Olle-Mineur** [reply]
I found it now! You needed to actually go into what component you want to translate and then you can start a new translation. I just went to the homepage of all languages and I saw now that you were in the correct component to create a new translation. 

![image](https://user-images.githubusercontent.com/48391622/114322568-3926c400-9b21-11eb-9a71-64a54e21fa9e.png)

**Reply by vabene1111** [reply]
ok good to kown. I just installed weblate yesterday so i am not very familar with it. I guess i will put a "how to start a new language" into the docs. 

Also let me know if there is anything you need help with so we can figure it out.
